lab-devon/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const readFile = require('./lib/read-files.js');
+
+const main = module.exports = function(argv) {
+  if(!argv || argv.length != 5) return 'Usage Error: not enough files were given';
+
+  return readFile(argv.slice(2,5), function(err,data){
+    if(err) return console.error(err);
+    console.log(data);
+  });
+};
+
+console.log(main(process.argv));
```

Comment:
You're not the only one who did this, and this doesn't break your code, but here's a copy/pasted comment I gave to Kenneth because had the same thing. 

========================

Just a small thing I wasn't aware of until I ran the code on my machine. Let's get rid of the ```console.log``` and just invoke the ```main``` function so that it says ```main(process.argv)```. 

When I ran your code in my terminal, I was first getting ```undefined``` and *then* the array of text output. If we just get rid of ```console.log``` and invoke ```main(process.argv)``` alone, we still get text output printed out because we already log the data on Line 12. 

The reason  for that is the console isn't able to log the output from ```main(process.argv)``` because the ```main``` function is async. 

In other words, ```console.log``` happens instantly while ```main(process.argv)``` is still going through the process of completing your ```readfiles``` function that happens on Line 8. 

```Console.log``` isn't going to wait until ```main(process.argv)``` is done, it's going to just print out contents right away, but since ```readFiles``` on Line 8 still hasn't completed, ```console.log``` spits out ```undefined.```

If you're confused, come find me and I'll try to re-explain again. :)